_data/simple-icons.json

```
@@ -8791,6 +8791,11 @@
                 "type": "MPL-2.0"
             }
         },
+        {
+            "title": "LibreTube",
+            "hex": "FFBBAA",
+            "source": "https://github.com/libre-tube/libre-tube.github.io/blob/e5e10090cab71ee7c0abdfbf2789977025733eb7/assets/icons/LibreTube.svg"
```

```suggestion
            "source": "https://github.com/libre-tube/libre-tube.github.io/blob/e5e10090cab71ee7c0abdfbf2789977025733eb7/assets/icons/icon.svg"
```

The icon source still matches (perfectly!) with your icon here, which I guess is the LibreTube icon minus the background of the source you have linked here now.